src/components/Checkbox/index.tsx

```diff
@@ -46,24 +56,35 @@ const StyledIcon = styled(Icon)`
 `
 
 const StyledChildrenContainer = styled('div', {
-  shouldForwardProp: prop => !['size'].includes(prop),
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
 })``
 
-const Checkbox = ({
-  checked,
-  onChange,
-  onFocus,
-  onBlur,
+type CheckboxProps = {
+  children: ReactNode
+  valid: boolean | undefined
+  error: string | ReactNode
+  size?: number
+  progress?: boolean
+  disabled?: boolean
+  typographyVariant?: string
+} & ReakitCheckboxProps &
+  XStyledProps
+
+const Checkbox: FunctionComponent<CheckboxProps> = ({
+  checked = false,
+  onChange = () => undefined,
```

Comment:
`onChange: PropTypes.func.isRequired,` in propTypes so there is no need for default value

Reply:
Default value is required for line 119: `if (!progress) onChange(e)`. Can't invoke undefined value. I fixed proptypes.

Reply:
Yes but the prop is required :)

Same for onFocus/Blur

Maybe change the type

```js
} & ReakitCheckboxProps &
```

```js
& ReakitCheckboxProps & Required<Pick<ReakitCheckboxProps, "onChange">> &
```